x/emissions/keeper/block_rewards_components.go

```diff
@@ -17,9 +17,9 @@ func (k Keeper) GetBlockRewardComponents(ctx sdk.Context) (sdk.Dec, sdk.Dec, sdk
 	return reservesFactor, bondFactor, durationFactor
 }
 func (k Keeper) GetBondFactor(ctx sdk.Context, stakingKeeper types.StakingKeeper) sdk.Dec {
-	targetBondRatio := sdk.MustNewDecFromStr(k.GetParams(ctx).TargetBondRatio)
-	maxBondFactor := sdk.MustNewDecFromStr(k.GetParams(ctx).MaxBondFactor)
-	minBondFactor := sdk.MustNewDecFromStr(k.GetParams(ctx).MinBondFactor)
+	targetBondRatio := sdk.MustNewDecFromStr(k.GetParamsIfExists(ctx).TargetBondRatio)
+	maxBondFactor := sdk.MustNewDecFromStr(k.GetParamsIfExists(ctx).MaxBondFactor)
+	minBondFactor := sdk.MustNewDecFromStr(k.GetParamsIfExists(ctx).MinBondFactor)
```

Comment:
Why dont we just remove these unused params.
https://github.com/zeta-chain/node/issues/1852

Reply:
I can remove this in this PR but overall we should address the whole investigation on what must be removed in this one: https://github.com/zeta-chain/node/issues/1813

Reply:
Looked at the code. I think it is better that we address all removal in the same PR.